h/static/styles/inject.scss

```diff
@@ -7,19 +7,18 @@
 $base-font-size: 14px;
 
 //ADDER////////////////////////////////
-.annotator-adder {
+.adder-group {
```

Comment:
See my note about whether we would just prefer this to stay as `annotator-adder`.